app/definitions/rest/v1/chat.ts

```diff
@@ -32,4 +32,11 @@ export type ChatEndpoints = {
 			total: number;
 		}>;
 	};
+	'chat.delete': {
+		POST: (params: { msgId: string; roomId: string }) => {
+			_id: string;
+			ts: Date;
```

Comment:
It's a date, you're correct, but the backend doesn't return an object, it's usually a string :)